### PR TITLE
ci(Smoke tests): RHICOMPL-1927 smoke tests to run on PR events to master

### PR DESCRIPTION
**Pull request by Victoremepunto**
Signed-off-by: Victor M <vmugicag@redhat.com>

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -46,7 +46,7 @@ def runStages() {
 
     scmVars = checkout scm
 
-    if (currentBuild.currentResult == "SUCCESS" && env.CHANGE_TARGET == "stable" && env.CHANGE_ID) {
+    if (currentBuild.currentResult == "SUCCESS" && env.CHANGE_TARGET == "master" && env.CHANGE_ID) {
```
Comment (akofink) [inline Jenkinsfile:49]: Could we get it to run on master and stable PRs for now while we still manually promote the backend to stable?

Comment (Victoremepunto) [reply Jenkinsfile]: sure thing, I added a conditional for both branch names

**Review by akofink — APPROVED** [review]
Thanks!

**Comment by akofink** [comment]
@Victoremepunto as you can see, the smoke test results will show but are currently not required (only required on merge to stable). Let's see how reliably green they are this week on PRs to master to see if we want to require it here as well.

**Comment by Victoremepunto** [comment]
> @Victoremepunto as you can see, the smoke test results will show but are currently not required (only required on merge to stable). Let's see how reliably green they are this week on PRs to master to see if we want to require it here as well.

Sure, but I think we should require them regardless - we're deprecating the `stable` approach so the idea would be to do quicker deployments of `master`. We can of course tune the smoke tests runs to select what gets run if it's too slow

**Pull request merged**